dock/router/router.py

```diff
@@ -59,43 +64,45 @@ def periodical_gossip(self):
             self.gossip(Chain(identifier=self.data_chain_id), target, 10, [])
             time.sleep(10)
 
-    def import_route(self):
-        with open(self.config_path) as file:
-            config = yaml.load(file, Loader=yaml.Loader)
-        route_path = os.path.join(os.path.dirname(self.config_path), config['router']['route_path'])
+    def configure(self, config_path):
+        with open(config_path) as file:
+            self.config = yaml.load(file, Loader=yaml.Loader)['router']
```

Comment:
Object member should be announced in __init__.

Reply:
why？

Reply:
Firstly, it's a good way to avoid call without init a member variables, besides it will triger warning. Secondly, a member variables init in a function may also confuse others while reading this code.

Reply:
Firstly, it is assigning a member without calling other member,besides there are lots of warnings in this project.
Secondly, the code I simplify the init function by this way is easy to read init function without reading useless code.

Reply:
I don’t think so, and it’s pointless.